src/Utils/request/uploadFile.ts

```
@@ -43,13 +50,61 @@
         handleHttpError(httpError);
         reject(httpError);
       } else {
+        if (
+          options &&
+          options.setOverallProgress &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)